Sources/AutomaticProperties.swift

```diff
@@ -117,8 +117,8 @@ class AutomaticProperties {
     }
     #endif
 
-    class func libVersion() -> String? {
-        return Bundle(for: self).infoDictionary?["CFBundleShortVersionString"] as? String
+    class func libVersion() -> String {
+        return "2.7.7"
```

Comment:
So this is going to be a hardcoded string here now?

Reply:
it will be changed dynamically via our releasing script. There is a problem reading from Info.plist in SPM environment, the Info.plist can't be copied to the actually framework bundle.